### ISSUE TITLE
Flow to update images

### ISSUE THREAD

**Issue by andriyun** [skilld-labs/docker-frontend#16]
To be sure what images have no regressions related to new changes need to create a script which will read current projects from config or arguments. For each project will do following steps:
1. Clone/update repo
2. Proceed/Build front task
3. Warn about failed builds

**Comment by andypost** [comment]
In #20 I refactored images so suggested flow is to use `frontend:latest` for testing & new features

`zen`, `d7` & omega should be supposed stable

The leftover question is how to test images - I guest we can always make in on projects in separate branch)

**Comment by andypost** [comment]
Here another idea, use https://hub.docker.com/r/skilldlabs/docker-frontend/ for testing images
So when testing went fine we manually push & deploy them to https://hub.docker.com/r/skilldlabs/frontend

**Comment by iberdinsky-skilld** [comment]
Actually i don't feel that often image update is cool in case of node modules especially. Images will became fat to cover every project requirements. New modules may use new dependencies and create bugs for old projects. 
Better to have something like https://github.com/skilld-labs/docker-frontend/pull/13 with basic images and project specific local update. So images update procedure became infrequent. And testing may be done manually. Flow suggestion:

1. Clone repo
2. Push on https://hub.docker.com/r/skilldlabs/docker-frontend/ 
3. Manually test on several listed in readme or whatever projects. 
4. Push on https://hub.docker.com/r/skilldlabs/frontend

Otherwise for testing we may have list of projects for every image tag. Script should clone them, run scripts in specific folders and report fails/success. What doesn't seems as easy, clear and stable procedure.

**Comment by andypost** [comment]
Vanya, you missed a point https://hub.docker.com/r/skilldlabs/docker-frontend/ is auto build

**Comment by andypost** [comment]
I totally disagree with concept where image will pull (install) packages in runtime
That's totally against the **concept of docker images** - they should carry everything inside

Instead of that we can create special image that will reuse local `node_modules` but this one exists - **mhart:alpine-node** it just needs to remove deletion of node_modules from your makefiles)
